client/views/signIn.jade

```diff
@@ -1,4 +1,4 @@
 template(name="signIn")
   .container.sign-in-wrap
     p Sign in or register to add a comment.
-      +atForm
\ No newline at end of file
+      +atForm
```

Comment:
I think this file can be removed. Looks like AccountsTemplate uses its own template?